scripts/variantstore/wdl/GvsExtractCohortFromSampleNames.wdl

```diff
@@ -0,0 +1,135 @@
+version 1.0
+
+import "GvsPrepareCallset.wdl" as GvsPrepareCallset
+import "GvsExtractCallset.wdl" as GvsExtractCallset
+
+# Workflow used by AoU to extract variants for a given cohort of sample_names
+
+workflow GvsExtractCohortFromSampleNames {
+
+  input {
+    File cohort_sample_names
+    String query_project
+    String gvs_dataset
+    String gvs_extraction_cohorts_dataset
+    String gvs_extraction_destination_dataset
+    String gvs_extraction_temp_tables_dataset
+    String extraction_uuid
+    String output_gcs_dir
+
+    # Extract parameters
+    File wgs_intervals
+    Int scatter_count
+
+    File reference
+    File reference_index
+    File reference_dict
+
+    String output_file_base_name
+
+    String? fq_filter_set_table
+    String? filter_set_name
+    File? gatk_override
+  }
+
+  call CreateCohortSampleTable {
+    input:
+      cohort_sample_names = cohort_sample_names,
+      gvs_dataset = gvs_dataset,
+      gvs_extraction_cohorts_dataset = gvs_extraction_cohorts_dataset,
+      query_project = query_project,
+      table_name = extraction_uuid
+  }
+
+  call GvsPrepareCallset.GvsPrepareCallset {
+    input:
+      destination_cohort_table_name   = extraction_uuid,
```

Comment:
What happens if the same dataset is used for `gvs_extraction_destination_dataset` and `gvs_extraction_cohorts_dataset`?  In both cases the table name is just the UUID so I think things will error out.  If that's the case maybe use the uuid as a suffix instead of the sole table name